### PR TITLE
Update cd.md

### PR DESCRIPTION
**Pull request by serragnoli**
Updating doc as per https://github.com/fastlane/fastlane/issues/15496#issuecomment-565028879

### CODE REVIEW

**Comment by legalcodes** [comment]
Nice catch @serragnoli , once the CLA is signed I think we're good to go. 

One question: the fastlane docs recommend:
```
Install fastlane using

# Using RubyGems
sudo gem install fastlane -NV
```

Have you verified that `gem install fastlane` without the flags and heightened permissions?

**Comment by serragnoli** [comment]
@googlebot I signed it!

**Comment by serragnoli** [comment]
Hi @legalcodes
Unfortunately I haven't tried the `gem` installation.
Thanks.

File: src/docs/deployment/cd.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ It's recommended that you test the build and deployment process locally before
 migrating to a cloud-based system. You could also choose to perform continuous
 delivery from a local machine.
 
-1. Install fastlane `gem install fastlane` or `brew cask install fastlane`.
+1. Install fastlane `gem install fastlane` or `brew install fastlane`.
```
Comment (legalcodes) [inline src/docs/deployment/cd.md]:
```suggestion
1. Install fastlane `gem install fastlane` or `brew install fastlane`.
Visit the [fastlane docs][fastlane] for more info.
```

**Comment by legalcodes** [comment]
@serragnoli OK, in that case I've added a link to the docs in this part. Thanks!

**Pull request merged**